internal/servers/job/jobs.go

```diff
@@ -0,0 +1,24 @@
+package servers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/boundary/internal/db"
+	"github.com/hashicorp/boundary/internal/kms"
+	"github.com/hashicorp/boundary/internal/scheduler"
+)
+
+// RegisterJobs registers rotate roots jobs with the provided scheduler.
```

Comment:
```suggestion
// RegisterJobs registers the rotate roots job with the provided scheduler.
```

Reply:
Alternatively if there will be other jobs in the future, the comment could just be generic about registering all required server jobs.